### DISCUSSION TITLE
Search in Fields Editor

### DISCUSSION THREAD

**Discussion by robinbeatty** [craftcms/cms#13961] [Ideas] [answered]
When developing, it's very common for us to be editing field types (cp > settings > fields) – could fields be searchable in this view? We find that we're often using the browser find text function to look for the field in lists of >100 field types.

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.0.0-alpha.9 was just released, with a search bar on the Fields and Entry Types index pages :tada:
